src/DotNetWorker.Grpc/RpcExtensions.cs

```diff
@@ -120,7 +121,7 @@ internal static FunctionDefinition ToFunctionDefinition(this FunctionLoadRequest
         /// <summary>
         /// Returns an RpcException for system exception scenarios.
         /// </summary>
-        /// <returns>An RpcException</returns>
+        /// <returns>An RpcException</returns> 
```

Comment:
nit: extra whitespace
```suggestion
        /// <returns>An RpcException</returns>
```